modules/ROOT/pages/errors/gql-errors.adoc

```diff
@@ -432,3 +432,264 @@ Status description:: error: syntax error or access rule violation - auth info ex
 === 42NFF
 
 Status description:: error: syntax error or access rule violation - permission/access denied. Access denied, see the security logs for details.
+
+== System configuration or operation exception
+
+System configuration or operation exception errors occur when there is an error in the system configuration or operation, such as procedure registration failure, a missing class field annotation, an unsupported injectable component type, duplicate field names, invalid map key type, etc.
+
+=== 51N00
+
+Status description:: error: system configuration or operation exception - procedure registration error. Failed to register procedure/function.
+
+
+=== 51N01
+
+Status description:: error: system configuration or operation exception - class field annotation should be public, non-final, and non-static. The field `{ $procField }` in the class `{ $procClass }` is annotated as a '@Context' field, but it is declared as static. '@Context' fields must be public, non-final and non-static.
+
+=== 51N02
+
+Status description:: error: system configuration or operation exception - unsupported injectable component type. Unable to set up injection for procedure `{ $procClass }`. The field `{ $procField }` has type `{ $procFieldType }` which is not a supported injectable component.
+
+
+=== 51N03
+
+Status description:: error: system configuration or operation exception - unable to access field. Unable to set up injection for `{ $procClass }`, failed to access field `{ $procField }`.
+
+
+=== 51N04
+
+Status description:: error: system configuration or operation exception - missing class field annotation. The field `{ $procField }` on `{ $procClass }` must be annotated as a '@Context' field in order to store its state.
+
+
+=== 51N05
+
+Status description:: error: system configuration or operation exception - class field should be public and non-final. The field `{ $procField }` on `{ $procClass }` must be declared non-final and public.
+
+
+=== 51N06
+
+Status description:: error: system configuration or operation exception - missing argument name. The argument at position `{ $pos }` in `{ $procMethod }` requires a '@Name' annotation and a non-empty name.
+
+
+=== 51N07
+
+Status description:: error: system configuration or operation exception - invalid ordering of default arguments. The `{ $procFun }` contains a non-default argument before a default argument. Non-default arguments are not allowed to be positioned after default arguments.
```

Comment:
```suggestion
Status description:: error: system configuration or operation exception - invalid ordering of default arguments. The `{ $procFun }` contains a non-default argument before a default argument. Non-default arguments are not allowed to be positioned before default arguments.
```

Reply:
I check with Gem and it is the other way around. The first sentence is wrong and the second correct so we can have 

```suggestion
Status description:: error: system configuration or operation exception - invalid ordering of default arguments. The `{ $procFun }` contains a non-default argument after a default argument. Non-default arguments are not allowed to be positioned after default arguments.
```

or maybe

```suggestion
Status description:: error: system configuration or operation exception - invalid ordering of default arguments. The `{ $procFun }` contains a default argument before a non-default argument. Non-default arguments are not allowed to be positioned after default arguments.
```